app/konnect/reference/audit-logs.md

```diff
@@ -6,17 +6,72 @@ content_type: reference
+| Authorization | Triggered when a permission check is made for a user or system account against a resource. | Not currently supported |
+| Access logs | Triggered when a request is made to the {{site.konnect_short_name}} API. | Not currently supported |
 
-## Data Retention Period
 {{site.konnect_short_name}} retains audit logs for 7 days. 
 
+## Audit log webhook status
+
+You can view the webhook status in the UI or via the API for the [{{site.konnect_short_name}} org audit logs](/konnect/org-management/audit-logging/webhook/) and [Dev Portal audit logs](/konnect/dev-portal/audit-logging/webhook/).
+
+The following table describes the webhook statuses:
+
+| Attribute | Description |
+| --------- | ---------- |
+| `last_attempt at` | The last time {{site.konnect_short_name}} tried to send data to your webhook |
+| `last_response_code` | The last response code from your webhook |
+| `webhook_enabled` | The desired status of the webhook (from `audit-log-webhook.enabled`) |
+| `webhook_status` | The actual status {{site.konnect_short_name}} of the webhook |
+
+A combination of `webhook_enabled` and `webhook_status` give a full picture of webhook status.
+
+| `webhook_enabled` | `webhook_status` | Description |
+| --------------- | -------------- | ---------- |
+| true            | `active`       | {{site.konnect_short_name}} is ready to send data to the webhook. Either no attempts have been made yet (`last_attempt_at` is not set), or the last attempt was successful. |
+| true            | `inactive`     | Last attempt to send data failed, but the webhook is still enabled. This usually means that there was an error in the endpoint or the SIEM provider went down that caused the logs to stop streaming. |
+| false           | `active`       | Webhook config is saved. {{site.konnect_short_name}} is not shipping data to it per webhook configuration. |
+| false           | `inactive`     |Last attempt to send data failed, and customer has turned off the webhook. |
+| false           | `unconfigured` | The webhook for this region has not been configured yet. |
+
 ## Log formats
 
 {{site.konnect_short_name}} delivers log events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm) or JSON. You may specify which format to use in the [audit log webhook](/konnect/org-management/audit-logging/webhook/) endpoint.
 
+The following are example audit logs in both formats:
+{% navtabs codeblock %}
+{% navtab CEF %}
+```
+Apr 14 05:39:08 konghq.com CEF:0|KongInc|Konnect|1.0|konnect|Authz.usage|1|rt=1681450748406 src=127.0.0.6 action=retrieve granted=true org_id=b065b594-6afc-4658-9101-5d9cf3f36b7b principal_id=87655c36-8d63-48fe-9a1e-53b28dfbc19b trace_id=3895213347334635099 user_agent=grpc-node/1.24.11 grpc-c/8.0.0 (linux; chttp2; ganges)
```

Comment:
The format differs slightly for DevPortal audit logs. This example represents what one might expect from a Konnect Org audit log, but I feel we should have example of DevPortal logs too and document the extra `portal_id` field.

I tried to capture this in a previous commit: https://github.com/Kong/docs.konghq.com/pull/7816/commits/b6d3aa9fca8eebb13aeb19e79677d1b9fe5e8795#diff-6c4dc5b26f5a4395618ddcf396cfbad8ceb081df139f65907ad604e85b52b519R104-R134